nemo_text_processing/text_normalization/normalize_with_audio.py

```diff
@@ -404,7 +404,7 @@ def parse_args():
         "--input_case", help="input capitalization", choices=["lower_cased", "cased"], default="cased", type=str
     )
     parser.add_argument(
-        "--language", help="Select target language", choices=["en", "ru", "de", "es"], default="en", type=str
+        "--language", help="Select target language", choices=["en", "ru", "de", "es", "sv"], default="en", type=str
```

Comment:
@jimregan was it intentional that "sv" is not added to deterministic normalize.py? if the default deterministic forms are reasonable, deterministic TN should be enabled as well. If you prefer to keep AB TN only, please add a similar commenthttps://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/normalize.py#L126.

Reply:
No, I just didn't see where to add it. I've found it since, thanks.